plugin/documents.py

```diff
@@ -228,22 +228,21 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
         if action_count == 0:
             return
         suffix = 's' if action_count > 1 else ''
```

Comment:
This variable can move into the `else` block.

Reply:
a1f5272